server/aws-lsp-codewhisperer/src/language-server/agenticChat/errors.ts

```diff
@@ -52,3 +53,16 @@ export function isInputTooLongError(error: unknown): boolean {
 
     return false
 }
+
+export function isRequestAbortedError(error: unknown): boolean {
+    if (error instanceof AgenticChatError && error.code === 'RequestAborted') {
+        return true
+    }
+
+    if (error instanceof Error) {
+        //  This is fragile (breaks if the backend changes their error message wording)
+        return error.message.includes('Request aborted')
```

Comment:
i cannot find this exception from RTS? which component does it from?

Reply:
The "Request aborted" error originates from the underlying fetch API used by the AWS SDK when a request is aborted.